OAK_Marketplace_Emotion_Recognition/utils/helpers.py

```diff
@@ -0,0 +1,16 @@
+import skvideo.io
```

Comment:
Create a file requirements.txt and put all dependencies there. Also, add `python3 -m pip install -r requirements.txt` to the installation guide. See this README for details - https://github.com/opencv-ai/opencv-blog/tree/main/PyTorch_ONNX_OpenVINO#requirements

Reply:
OK. Probably don't need to do it because modelplace-api requires this packages https://github.com/opencv-ai/modelplace-api/blob/main/setup.py